openml_OS/models/api/v1/Api_data.php

```diff
@@ -307,12 +337,10 @@ private function data_delete($data_id) {
       //}
     }
 
-    $result = $this->Dataset->delete( $dataset->did );
-    $this->Data_feature->deleteWhere('did =' . $dataset->did);
-    $this->Data_quality->deleteWhere('data =' . $dataset->did);
+    $result = $this->Dataset->delete($dataset->did);
```

Comment:
Why not delete the data features and qualities? Is deletion guaranteed by mysql cascades?

Reply:
yes